powerlevel9k.zsh-theme

```diff
@@ -1317,8 +1317,8 @@ prompt_powerlevel9k_setup() {
 
   # Display a warning if the terminal does not support 256 colors
   local term_colors
-  term_colors=$(echotc Co)
-  if (( $term_colors < 256 )); then
+  term_colors=$(echotc Co 2>/dev/null)
+  if (( ! $? && ${term_colors:-0} < 256 )); then
```

Comment:
I am not sure if that will bring the desired clarity. I'd like to hear some other people on this (ping @bhilburn ).
And probably a comment will do. ;)